front-end/src/App.tsx

```diff
@@ -36,6 +37,7 @@ function App() {
               {loggedInUser && <Route path={`/author/${loggedInUser.authorId}`} render={(props) => <AuthorPage {...props} loggedInUser={loggedInUser} />} />}
               {/* TODO: hide settings page if not logged in */}
               <Route path="/settings" render={(props) => <SettingsPage loggedInUser={loggedInUser} />} />
+              <Route path="/author_detail/:id" render={(props) => <AuthorDetail {...props}/>} />
```

Comment:
The route should just be `"/author/:id/"`, which already exists. Right now we should be able to search ourselves up (i.e., if you are logged in, you should be able to search for yourself and click on your page) but it will just show a "404" if you try to view any other author